src/main/java/com/appland/appmap/transform/annotations/Hook.java

```diff
@@ -204,6 +199,45 @@ public static void apply(CtBehavior targetBehavior, List<HookSite> hookSites) {
     }
   }
 
+  /* Concatenates potentially null strings with no delimeter. The return value
+     is guaranteed to be non-null.
+  */
+  private static String safeConcatStrings(String... strs) {
```

Comment:
Maybe I've just spent too much time writing Ruby, but this looks gross to me.

How would you feel about:
```
    return Arrays.stream(strs)
      .filter(Objects::nonNull)
      .reduce("", (memo, s) -> memo + s.toString());
  }
```